Scripts/Editor/NodeGraphEditor.cs

```diff
@@ -42,7 +42,7 @@ public class NodeGraphEditor : XNodeEditor.Internal.NodeEditorBase<NodeGraphEdit
         }
 
         /// <summary> Add items for the context menu when right-clicking this node. Override to add custom menu items. </summary>
-        public virtual void AddContextMenuItems(GenericMenu menu) {
+        public virtual void AddContextMenuItems(GenericMenu menu, GenericMenu.MenuFunction2 call = default(GenericMenu.MenuFunction2)) {
```

Comment:
This new parameter `GenericMenu.MenuFunction2 call = default(GenericMenu.MenuFunction2` breaks compatability and is very confusing. This looks like a dirty workaround.

Reply:
Um.. yeah your kinda right, but i need a way to call it when i want to since i dont want to call every time the function is called

Reply:
@Siccity nevermind fixed the problem, the annoying parameter is deleted.